src/iterative/KOSMAPOSL/KOSMAPOSLReconstruction.cxx

```diff
@@ -300,8 +258,67 @@ set_up(shared_ptr <TargetT > const& target_image_ptr)
 {
 
  if (base_type::set_up(target_image_ptr) == Succeeded::no)
-     error("KOSMAPOSL::set_up(): Error building post filter");
-
+     error("KOSMAPOSL::set_up(): Error setting-up underlying OSMAPOSLReconstruction object");
+
+ if (is_null_ptr(this->anatomical_prior_sptrs[0]) &&
+     this->sigma_m.size()==0 )
```

Comment:
this one still remains. the rest looks good to me.
```suggestion
if ((this->anatomical_prior_sptrs.size()==0) &&
     (this->hybrid==0))
```

Reply:
sorry for some reason I must have pressed ctrl+z and changed back.

Reply:
still not changed the first line to `(this->anatomical_prior_sptrs.size()==0)` I'm afraid. Editor playing up...